bedrock/newsletter/tests/test_views.py

```diff
@@ -646,14 +655,14 @@ def test_returns_failure__invalid_newsletter(self, basket_mock):
     def test_returns_failure__missing_privacy(self, basket_mock):
         """Test non-ajax POST with missing privacy acceptance."""
         data = {
-            "newsletters": "flintstones",
+            "newsletters": ["flintstones"],
             "email": "fred@example.com",
             "fmt": "H",
         }
         resp = self.request(data=data)
         doc = pq(resp.content)
         self.assertTrue(doc("#newsletter-form"))
-        self.assertTrue(doc('input[value="mozilla-and-you"]')[0].checked)
+        self.assertTrue(doc('input[value="flintstones"]')[0].checked)
```

Comment:
Have we lost the fallback to a common/populat newsletter?

Reply:
There is no check on what is an existing newsletter so it accepts whatever string you pass to it as long as it is a valid slug. In this test, `flintstones` is the default fallback since it is a valid newsletter slug. Versus another test that uses `!nv@lid` which then does fall back to a known default.